internal/datastore/crdb/caveat.go

```diff
@@ -22,7 +22,7 @@ var (
 	)
 	writeCaveat  = psql.Insert(tableCaveat).Columns(colCaveatName, colCaveatDefinition).Suffix(upsertCaveatSuffix)
 	readCaveat   = psql.Select(colCaveatDefinition, colTimestamp).From(tableCaveat)
-	listCaveat   = psql.Select(colCaveatName, colCaveatDefinition).From(tableCaveat).OrderBy(colCaveatName)
+	listCaveat   = psql.Select(colCaveatName, colCaveatDefinition, colTimestamp).From(tableCaveat).OrderBy(colCaveatName)
```

Comment:
do we need a new index for this? 👀

Reply:
I don't believe so; the name is the primary key and we're only selecting by name in the hot path (ListCaveats is returning everything)

Reply:
but then it won't be a covering index and will require access to disk - just want to make sure are being intentional about this